script/postpublish

```diff
@@ -40,9 +40,11 @@ if demo_package['dependencies']['amazon-chime-sdk-js'] != "^#{npm_version}"
   exit(1)
 end
 
-meeting_app = 'app/meetingV2/meetingV2.ts'
+meeting_app = 'app/meeting/meeting.ts'
+meetingV2_app = 'app/meetingV2/meetingV2.ts'
 
 File.write(meeting_app, File.read(meeting_app).gsub("'../../../../src/index'", "'amazon-chime-sdk-js'"))
+File.write(meetingV2_app, File.read(meetingV2_app).gsub("'../../../../src/index'", "'amazon-chime-sdk-js'"))
 verbose('npm run build')
```

Comment:
I think we need to build twice since the `serverless` deploy script does not re-build them.
```
npm run start --app=meeting
npm run start --app=meetingV2
```